vignettes/varying_susceptibility.Rmd

```diff
@@ -51,11 +51,25 @@ knitr::opts_chunk$set(
 )
 ```
 
+## Primer on heterogeneous susceptibility
```

Comment:
Thanks also for adding the suggested paragraphs in this new section. However, as I commented here https://github.com/epiverse-trace/finalsize/pull/121/files#r1074004317, I think that this is not precise for the topic. Also, I prefer to not relate susceptibility with prevalences, as refered in this comment https://github.com/epiverse-trace/finalsize/issues/116#issuecomment-1385926597

As an alternative, in this section we can define "susceptibility", and add text from the references that I used in the first comment, explaining the age-specific factors for the between-difference. This still will work as a primer.

In contrast to the within-differences, as far as I understood, the between-differences can not be directly gathered from data, but taken into consideration in the modelling by having these articles as references.

**Here is a new suggested text:**

"We can define susceptibility as the probability of becoming infected on contact with an infectious person.

Age-specific factors influencing the susceptibility of individuals include, but are not limited to, direct (immunological) susceptibility to infection upon exposure (e.g., due to age-specific heterogeneous risk behaviour or compliance to non-pharmaceutical interventions not already captured by contact frequency, natural susceptibility from previous infections, differences in vaccination status, etc.) ([Franco et al., 2022](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9000131/))

For example, for SARS-CoV-2, children are less susceptible to it than adults. Decreased susceptibility could result from [immune cross-protection from other coronaviruses](https://www.nature.com/articles/s41467-020-18450-4), or from non-specific protection resulting from recent infection by [other respiratory viruses](https://academic.oup.com/cid/article/54/12/1778/455098?login=false), which children experience [more frequently than adults](https://journals.lww.com/md-journal/Fulltext/2018/06010/Age_related_prevalence_of_common_upper_respiratory.45.aspx). ([Davies et al., 2020](https://www.nature.com/articles/s41591-020-0962-9)) "


Here is as plain text:
```
We can define susceptibility as the probability of becoming infected on contact with an infectious person.

Age-specific factors influencing the susceptibility of individuals include, but are not limited to, direct (immunological) susceptibility to infection upon exposure (e.g., due to age-specific heterogeneous risk behaviour or compliance to non-pharmaceutical interventions not already captured by contact frequency, natural susceptibility from previous infections, differences in vaccination status, etc.) ([Franco et al., 2022](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9000131/))

For example, for SARS-CoV-2, children are less susceptible to it than adults. Decreased susceptibility could result from [immune cross-protection from other coronaviruses](https://www.nature.com/articles/s41467-020-18450-4), or from non-specific protection resulting from recent infection by [other respiratory viruses](https://academic.oup.com/cid/article/54/12/1778/455098?login=false), which children experience [more frequently than adults](https://journals.lww.com/md-journal/Fulltext/2018/06010/Age_related_prevalence_of_common_upper_respiratory.45.aspx). ([Davies et al., 2020](https://www.nature.com/articles/s41591-020-0962-9))
```

Reply:
I'm adding this text to be relatively general about the mechanisms behind heterogeneous susceptibility; the references should be linked automatically as in `vignettes/finalsize.Rmd`.

> A working definition of susceptibility is the probability of becoming infected on contact with an infectious person.
> 
> Age-specific factors influencing the susceptibility of individuals include, but are not limited to, direct (immunological) susceptibility to infection upon exposure (e.g., due to natural susceptibility from previous infections, differences in vaccination status), and age-specific heterogeneous risk behaviour [@franco2022].
> 
> For example, for SARS-CoV-2, children are less susceptible to it than adults. Decreased susceptibility could result from immune cross-protection from other coronaviruses [@huang2020], or from non-specific protection resulting from recent infection by other respiratory viruses [@cowling2012], which children experience more frequently than adults [@tsagarakis2018, @davies2020].

Reply:
Updated in https://github.com/epiverse-trace/finalsize/pull/121/commits/dc474e6f0beb103d016d14812904855f1b00f054